lib/async.js

```diff
@@ -46,7 +46,7 @@ module.exports = function resolve (x, opts, cb) {
     else loadNodeModules(x, y, function (err, n, pkg) {
         if (err) cb(err)
         else if (n) cb(null, n, pkg)
-        else if (core[x]) return cb(null, x);
+        else if (core[x]) return cb(null, x, null);
```

Comment:
this is switching it from calling with `undefined` to calling with `null` - I'm not sure why this is an improvement?